ruamel_yaml/__init__.py

```diff
@@ -1,7 +1,7 @@
 # coding: utf-8
 from __future__ import print_function, absolute_import
 
-__version__ = "0.11.7"
+__version__ = "$VERSION"
```

Comment:
Not strictly required. However, the `sed` expression would already replace this line with this change. Plus this makes it more clear to the end user what is happening with this version info. Namely it is being patched out with the contents of the `$VERSION` environment variable.